src/Data/Vec/Functional/Algebra/Properties.agda

```diff
@@ -0,0 +1,725 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties for the functional vector algebra approach
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Vec.Functional.Algebra.Properties where
+
+open import Level using (Level; _⊔_)
+open import Function using (_$_; flip)
+open import Data.Product hiding (map)
+open import Data.Nat using (ℕ)
+open import Data.Fin using (Fin; zero; suc)
+open import Function.Base using (_∘′_; _ˢ_; _∘_)
+open import Data.Vec.Functional
+open import Algebra.Core
+open import Algebra.Bundles
+open import Algebra.Module
+open import Algebra.Module.Structures
+open import Relation.Binary
+import Algebra.Definitions as Definitions
+open import Algebra.Structures
+open import Data.Vec.Functional.Algebra.Base
+import Data.Vec.Functional.Relation.Binary.Pointwise.Properties as Pointwise
+
+private variable
+  a ℓ : Level
+  A : Set ℓ
+  n : ℕ
+
+------------------------------------------------------------------------
+-- Raw structure properties
+
+module RawMagmaProperties (rawMagma : RawMagma a ℓ) (n : ℕ) where
+  private
+    magmaᵛ = rawMagmaᵛ rawMagma n
+    module VM = RawMagma magmaᵛ
+    module M = RawMagma rawMagma
+
+  open IsEquivalence
+
+  isMagmaᵛ : IsMagma M._≈_ M._∙_ → IsMagma VM._≈_ VM._∙_
+  isMagmaᵛ base = record
+    { isEquivalence = flip Pointwise.isEquivalence _ B.isEquivalence
+    ; ∙-cong        = λ x≈y u≈v i → B.∙-cong (x≈y i) (u≈v i)
+    }
+    where module B = IsMagma base
```

Comment:
Now you've chosen `base` for the 'base' argument, you won't need the superscript...?

As for `module B = IsMagma base` why change the names `module CM = ...`, `module SG = ...` of the subsequent locally-bound modules... when `B` would suffice in those instances as well?